src/lib.rs

```diff
@@ -194,6 +194,34 @@
 //! # fn main() {}
 //! ```
 //!
+//! ## Hidden Fields
+//!
+//! You can hide fields by skipping their setters on the builder struct.
+//!
+//! - Opt-out &mdash; skip setters via `#[builder(setter(skip))]` on individual fields.
+//! - Opt-in &mdash; set `#[builder(setter(skip))]` on the whole struct
```

Comment:
yes, `&mdash;` works in rustdoc - afaik it's a feature of markdown in general. :-)